Ошибка при скачивании треков с метаданными

**Опишите сам баг**
При скачивании _некоторых_ треков со включённой настройкой "Сохранять метадату песен" вылетает ошибка по окончании загрузки.

**Опишите действия для воспроизведения ошибки**
1. Найти трек, с которого нельзя перейти напрямую к карточке исполнителя через три точки -> "Перейти к исполнителю" (судя по всему, проблема именно с такими треками). [Пример](https://vk.com/audio281039044_456241062_b8a5fb771033ca9ed3)
2. Три точки -> "Скачать как mp3"
3. Дождаться конца загрузки

**Опишите ожидаемый результат**
Трек скачивается, (в идеале) в нём есть метаданные

**Скриншот ошибки**
![Screenshot_2023-10-22-23-03-30-225_com vtosters lite](https://github.com/vtosters/lite/assets/96656876/31713e43-a144-4c5b-8afe-3c2437607568)

**Информация об устройстве:** 

- Commit: f2ac845
- Full Version Information: main-f2ac845-dirty
- Android SDK: 30
- Product: ginkgo
- Device: ginkgo
- Board: ginkgo
- Manufacturer: Xiaomi
- Brand: xiaomi
- Model: Redmi Note 8
- Miui Version Code: 12
- Miui Version Name: V125
- Miui Version Incremental Code: V12.5.2.0.RCOMIXM

**Дополнительная информация о баге или мыслях почему оно может происходить (если есть)**
Насколько я понял, у вышеуказанных треков в полях `MusicTrack.L` (список исполнителей) и `MusicTrack.I` (альбом) лежит `null`, из-за этого в `FFMpeg.setMetadata` бросается `NullPointerException`. Если сделать там проверку на `null`, то метаданных всё ещё не будет, но по крайней мере трек будет сохраняться

Кусок VK.log-main.log из Dev-билда:
```
10-22 20:21:53.404 E/M3UDownloader( 8457): Failed to download track
10-22 20:21:53.404 E/M3UDownloader( 8457): java.lang.RuntimeException: FFmpeg error
10-22 20:21:53.404 E/M3UDownloader( 8457):  at ru.vtosters.lite.music.downloader.M3UDownloader.lambda$parse$2(M3UDownloader.java:71)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at ru.vtosters.lite.music.downloader.M3UDownloader$$ExternalSyntheticLambda3.run(Unknown Source:6)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:778)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:511)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1793)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.FJTask.doExec(FJTask.java:271)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.FJPool.runWorker(FJPool.java:1495)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at java8.util.concurrent.FJWorkerThread.run(FJWorkerThread.java:122)
10-22 20:21:53.404 E/M3UDownloader( 8457): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Class java.lang.Object.getClass()' on a null object reference
10-22 20:21:53.404 E/M3UDownloader( 8457):  at j$.util.Objects.requireNonNull(Unknown Source:0)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at j$.util.Collection$-EL.spliterator(SourceFile:0)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at j$.util.Map$-CC.$default$stream(Unknown Source:0)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at j$.util.Collection$-EL.stream(Unknown Source:11)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at ru.vtosters.lite.music.converter.ts.FFMpeg.setMetadata(FFMpeg.java:37)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at ru.vtosters.lite.music.converter.ts.FFMpeg.convert(FFMpeg.java:29)
10-22 20:21:53.404 E/M3UDownloader( 8457):  at ru.vtosters.lite.music.downloader.M3UDownloader.lambda$parse$2(M3UDownloader.java:69)
10-22 20:21:53.404 E/M3UDownloader( 8457):  ... 8 more
```

--- a/app/src/main/java/ru/vtosters/lite/music/converter/ts/FFMpeg.java
+++ b/app/src/main/java/ru/vtosters/lite/music/converter/ts/FFMpeg.java
@@ -33,9 +33,14 @@ public class FFMpeg {
     }
 
     private static String setMetadata(MusicTrack track) {
-        return Preferences.getBoolValue("setMetaData", false) ? " -metadata title=\"" + IOUtils.getValidFileName(M3UDownloader.getTitle(track)) + '"'
-                + " -metadata artist=\"" + normalizeMetadata(track.L.stream().map(Artist::w1).collect(Collectors.joining(", "))) + '"'
-                + " -metadata album=\"" + normalizeMetadata(track.I.getTitle()) + '"' : "";
+        if (!Preferences.getBoolValue("setMetaData", false)) {
+            return "";
+        }
+        return " -metadata title=\"" + IOUtils.getValidFileName(M3UDownloader.getTitle(track)) + '"'
+                + (track.L == null || track.I == null
+                ? " -metadata artist=\"" + normalizeMetadata(track.C) + '"'
+                : " -metadata artist=\"" + normalizeMetadata(track.L.stream().map(Artist::w1).collect(Collectors.joining(", "))) + '"'
+                + " -metadata album=\"" + normalizeMetadata(track.I.getTitle()) + '"');
     }
 
     private static String normalizeMetadata(String in) {

